### ISSUE TITLE
Did you know about Squoosh.net?

### ISSUE THREAD

**Issue by johnnyreilly** [GoogleChromeLabs/squoosh#1285]
Heya,

Not a bug report - but rather a report of something surprising/possibly nefarious. I discovered that there's a squoosh.net site out there that looks very similar to squoosh.app but appears to be something different.

I don't know if you're aware of this, but it struck me as peculiar and potentially worrying. I'm guessing there's nothing to do about this, but people should probably be aware if this is malign. Apologies if I'm just creating pointless noise

**Comment by surma** [comment]
It’s a Chinese translation for Squoosh v1. 

Is there anything specifically that you makes you worry or call it malign?

**Comment by johnnyreilly** [comment]
Ah my apologies. I worried because it was not squoosh.app address wise and it looked slightly different from squoosh.app when I compared the two. Given Squoosh creates files, I wondered if this was someone using nearly the same domain, but not quite and perhaps generating files which contained something unwanted. 

When I noticed that squoosh.net was not squoosh.app I closed the browser tab and didn't go back.

My apologies for the false alarm.
